app/controllers/application_records_controller.rb

```diff
@@ -15,6 +15,9 @@ def create
                                                           user: @current_user,
                                                           reviewed: false))
     record.email_subscribers applicant: @current_user
+
+    create_unavailability(record) if params[:unavailability_enabled] == 'true'
```

Comment:
Shouldn't need to pass this as a parameter - it's an attribute of the application template in question.